omod/src/main/java/org/openmrs/web/controller/PortletController.java

```diff
@@ -265,6 +265,38 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 					} else {
 						model.put("patientObs", new HashSet<Obs>());
 					}
+					/**
+					 * Copied from OpenMRS core 1.9.x
+					 * See https://github.com/openmrs/openmrs-core/blob/1.9.x/web/src/main/java/org/openmrs/web/controller/PortletController.java#L267
```

Comment:
A line from 1.9.x may point to something very different, a few years from now. You need to use a tag.

Reply:
[Fixed](https://github.com/openmrs/openmrs-module-legacyui/pull/128/commits/9dd92c8a5560377daedb2e231c639e315fa177b8)